pkgs/development/python-modules/apptools/default.nix

```diff
@@ -26,11 +26,20 @@ buildPythonPackage rec {
   };
 
   patches = [
+    # python310: Fix tests
+    # https://github.com/enthought/apptools/issues/303
+    (fetchpatch {
+      url = "https://github.com/enthought/apptools/commit/0d65869701b3bab49d253c5489002664316b32fb.patch";
```

Comment:
https://github.com/enthought/apptools/commit/10fb73916124f7ae7edf6c6688a05ad95678488f is the commit that actually made it into upstream - think they probably did a rebase-merge of the PR.

Reply:
I updated the fetchpatch commit to the one being merged upstream :)